lightkurve/correctors/designmatrix.py

```diff
@@ -272,6 +272,24 @@ def append_constant(self, prior_mu=0, prior_sigma=np.inf, inplace=False):
         dm.prior_sigma = np.append(self.prior_sigma, prior_sigma)
         return dm
 
+    def mask(self, cadence_mask):
```

Comment:
Do you think we *need* to add this new method?  It doesn't appear to be used by any new code in this PR.

This function is somewhat inconsistent with LightCurve and TargetPixelFile objects, which can only be masked using rectangular brackets rather than a mask method, so my preference is not to introduce this method unless there is a compelling reason to do so?

Reply:
You're right, this was used in the now-removed GP functions and was added for clarity. I'll remove it.